[hyperscan] build failure

### Operating system

Linux (Ubuntu 22.04)

### Compiler

GCC

### Steps to reproduce the behavior

```Shell
`./vcpkg install hyperscan`
```


### Failure logs

```
username@hostname:~/src/todds/build$ cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=./ -DCMAKE_TOOLCHAIN_FILE=../vcpkg/scripts/buildsystems/vcpkg.cmake -DVCPKG_TARGET_TRIPLET=x64-linux ..
-- Running vcpkg install
Detecting compiler hash for triplet x64-linux...
The following packages will be built and installed:
  * at-spi2-atk[core]:x64-linux -> 2.38.0
  * at-spi2-core[core]:x64-linux -> 2.44.1#2
  * atk[core]:x64-linux -> 2.38.0#5
  * boost-algorithm[core]:x64-linux -> 1.82.0#1
  * boost-align[core]:x64-linux -> 1.82.0#1
  * boost-any[core]:x64-linux -> 1.82.0#1
  * boost-array[core]:x64-linux -> 1.82.0#1
  * boost-assert[core]:x64-linux -> 1.82.0#1
  * boost-atomic[core]:x64-linux -> 1.82.0#1
  * boost-bimap[core]:x64-linux -> 1.82.0#1
  * boost-bind[core]:x64-linux -> 1.82.0#1
  * boost-build[core]:x64-linux -> 1.82.0#1
  * boost-chrono[core]:x64-linux -> 1.82.0#1
  * boost-circular-buffer[core]:x64-linux -> 1.82.0#1
  * boost-concept-check[core]:x64-linux -> 1.82.0#1
  * boost-config[core]:x64-linux -> 1.82.0#1
  * boost-container[core]:x64-linux -> 1.82.0#1
  * boost-container-hash[core]:x64-linux -> 1.82.0#1
  * boost-conversion[core]:x64-linux -> 1.82.0#1
  * boost-core[core]:x64-linux -> 1.82.0#1
  * boost-crc[core]:x64-linux -> 1.82.0#1
  * boost-date-time[core]:x64-linux -> 1.82.0#1
  * boost-describe[core]:x64-linux -> 1.82.0#1
  * boost-detail[core]:x64-linux -> 1.82.0#1
  * boost-dynamic-bitset[core]:x64-linux -> 1.82.0#1
  * boost-endian[core]:x64-linux -> 1.82.0#1
  * boost-exception[core]:x64-linux -> 1.82.0#1
    boost-filesystem[core]:x64-linux -> 1.82.0#1
  * boost-foreach[core]:x64-linux -> 1.82.0#1
  * boost-format[core]:x64-linux -> 1.82.0#1
  * boost-function[core]:x64-linux -> 1.82.0#1
  * boost-function-types[core]:x64-linux -> 1.82.0#1
  * boost-functional[core]:x64-linux -> 1.82.0#1
  * boost-fusion[core]:x64-linux -> 1.82.0#1
  * boost-graph[core]:x64-linux -> 1.82.0#1
  * boost-icl[core]:x64-linux -> 1.82.0#1
  * boost-integer[core]:x64-linux -> 1.82.0#1
  * boost-intrusive[core]:x64-linux -> 1.82.0#1
  * boost-io[core]:x64-linux -> 1.82.0#1
  * boost-iterator[core]:x64-linux -> 1.82.0#1
  * boost-lambda[core]:x64-linux -> 1.82.0#1
  * boost-lexical-cast[core]:x64-linux -> 1.82.0#1
  * boost-math[core]:x64-linux -> 1.82.0#1
  * boost-modular-build-helper[core]:x64-linux -> 1.82.0#1
  * boost-move[core]:x64-linux -> 1.82.0#1
  * boost-mp11[core]:x64-linux -> 1.82.0#1
  * boost-mpl[core]:x64-linux -> 1.82.0#1
  * boost-multi-array[core]:x64-linux -> 1.82.0#1
  * boost-multi-index[core]:x64-linux -> 1.82.0#1
    boost-nowide[core]:x64-linux -> 1.82.0#1
  * boost-numeric-conversion[core]:x64-linux -> 1.82.0#1
  * boost-optional[core]:x64-linux -> 1.82.0#1
  * boost-parameter[core]:x64-linux -> 1.82.0#1
  * boost-phoenix[core]:x64-linux -> 1.82.0#1
  * boost-pool[core]:x64-linux -> 1.82.0#1
  * boost-predef[core]:x64-linux -> 1.82.0#1
  * boost-preprocessor[core]:x64-linux -> 1.82.0#1
  * boost-property-map[core]:x64-linux -> 1.82.0#1
  * boost-property-tree[core]:x64-linux -> 1.82.0#1
  * boost-proto[core]:x64-linux -> 1.82.0#1
  * boost-ptr-container[core]:x64-linux -> 1.82.0#1
  * boost-random[core]:x64-linux -> 1.82.0#1
  * boost-range[core]:x64-linux -> 1.82.0#1
  * boost-ratio[core]:x64-linux -> 1.82.0#1
  * boost-rational[core]:x64-linux -> 1.82.0#1
  * boost-regex[core]:x64-linux -> 1.82.0#1
  * boost-serialization[core]:x64-linux -> 1.82.0#1
  * boost-smart-ptr[core]:x64-linux -> 1.82.0#1
  * boost-spirit[core]:x64-linux -> 1.82.0#1
  * boost-static-assert[core]:x64-linux -> 1.82.0#1
  * boost-system[core]:x64-linux -> 1.82.0#1
  * boost-thread[core]:x64-linux -> 1.82.0#1
  * boost-throw-exception[core]:x64-linux -> 1.82.0#1
  * boost-tokenizer[core]:x64-linux -> 1.82.0#1
  * boost-tti[core]:x64-linux -> 1.82.0#1
  * boost-tuple[core]:x64-linux -> 1.82.0#1
  * boost-type-index[core]:x64-linux -> 1.82.0#1
  * boost-type-traits[core]:x64-linux -> 1.82.0#1
  * boost-typeof[core]:x64-linux -> 1.82.0#1
  * boost-uninstall[core]:x64-linux -> 1.82.0#1
  * boost-unordered[core]:x64-linux -> 1.82.0#1
  * boost-utility[core]:x64-linux -> 1.82.0#1
  * boost-variant[core]:x64-linux -> 1.82.0#1
  * boost-variant2[core]:x64-linux -> 1.82.0#1
  * boost-vcpkg-helpers[core]:x64-linux -> 1.82.0#1
  * boost-winapi[core]:x64-linux -> 1.82.0#1
  * boost-xpressive[core]:x64-linux -> 1.82.0#1
  * brotli[core]:x64-linux -> 1.0.9#5
  * bzip2[core,tool]:x64-linux -> 1.0.8#4
  * cairo[core,fontconfig,freetype,gobject,x11]:x64-linux -> 1.17.8
  * dirent[core]:x64-linux -> 1.23.2#2
  * egl-registry[core]:x64-linux -> 2022-09-20
  * expat[core]:x64-linux -> 2.5.0#3
    fmt[core]:x64-linux -> 9.1.0#1
  * fontconfig[core]:x64-linux -> 2.14.2
  * freetype[brotli,bzip2,core,png,zlib]:x64-linux -> 2.12.1#3
  * fribidi[core]:x64-linux -> 1.0.12
  * gdk-pixbuf[core]:x64-linux -> 2.42.10
  * getopt[core]:x64-linux -> 0#3
  * gettext[core,tools]:x64-linux -> 0.21.1
  * glib[core]:x64-linux -> 2.76.1
  * gperf[core]:x64-linux -> 3.1#5
  * gtk3[core]:x64-linux -> 3.24.34#2
  * harfbuzz[core]:x64-linux -> 7.1.0
    hwloc[core]:x64-linux -> 2.9.0
    hyperscan[core]:x64-linux -> 5.4.0
  * libepoxy[core]:x64-linux -> 1.5.10#1
  * libffi[core]:x64-linux -> 3.4.4#1
  * libiconv[core]:x64-linux -> 1.17#1
  * libjpeg-turbo[core]:x64-linux -> 2.1.5.1
  * liblzma[core]:x64-linux -> 5.4.1#1
  * libpng[core]:x64-linux -> 1.6.39#1
  * libuuid[core]:x64-linux -> 1.0.3#12
  * libwebp[core,libwebpmux,nearlossless,simd]:x64-linux -> 1.3.0
  * libxml2[core,iconv,lzma,zlib]:x64-linux -> 2.10.3#1
  * lzo[core]:x64-linux -> 2.10#9
    opencv[core,default-features]:x64-linux -> 4.7.0#1
  * opencv4[core,default-features,dnn,gtk,jpeg,png,quirc,tiff,webp]:x64-linux -> 4.7.0#5
  * pango[core]:x64-linux -> 1.50.14
  * pcre[core]:x64-linux -> 8.45#5
  * pcre2[core]:x64-linux -> 10.40#1
  * pixman[core]:x64-linux -> 0.42.2#2
  * protobuf[core]:x64-linux -> 3.21.12
  * pthread[core]:x64-linux -> 3.0.0#2
  * pthreads[core]:x64-linux -> 3.0.0#13
  * quirc[core]:x64-linux -> 1.1#3
  * ragel[core]:x64-linux -> 6.10#6
    tbb[core]:x64-linux -> 2021.8.0
  * tiff[core,jpeg,lzma,zip]:x64-linux -> 4.5.0#2
  * vcpkg-cmake[core]:x64-linux -> 2022-12-22
  * vcpkg-cmake-config[core]:x64-linux -> 2022-02-06#1
  * vcpkg-cmake-get-vars[core]:x64-linux -> 2023-03-02
  * vcpkg-get-python-packages[core]:x64-linux -> 2022-06-30
  * vcpkg-tool-meson[core]:x64-linux -> 0.63
  * zlib[core]:x64-linux -> 1.2.13
Additional packages (*) will be modified to complete this operation.
Restored 91 package(s) from /home/username/.cache/vcpkg/archives in 704 ms. Use --debug to see more details.
Installing 1/135 vcpkg-cmake:x64-linux...
Elapsed time to handle vcpkg-cmake:x64-linux: 804 us
Installing 2/135 boost-uninstall:x64-linux...
Elapsed time to handle boost-uninstall:x64-linux: 477 us
Installing 3/135 boost-vcpkg-helpers:x64-linux...
Elapsed time to handle boost-vcpkg-helpers:x64-linux: 480 us
Installing 4/135 boost-predef:x64-linux...
Elapsed time to handle boost-predef:x64-linux: 8.26 ms
Installing 5/135 boost-config:x64-linux...
Elapsed time to handle boost-config:x64-linux: 10.1 ms
Installing 6/135 boost-winapi:x64-linux...
Elapsed time to handle boost-winapi:x64-linux: 5.07 ms
Installing 7/135 boost-static-assert:x64-linux...
Elapsed time to handle boost-static-assert:x64-linux: 647 us
Installing 8/135 boost-type-traits:x64-linux...
Elapsed time to handle boost-type-traits:x64-linux: 13.2 ms
Installing 9/135 boost-assert:x64-linux...
Elapsed time to handle boost-assert:x64-linux: 859 us
Installing 10/135 boost-throw-exception:x64-linux...
Elapsed time to handle boost-throw-exception:x64-linux: 780 us
Installing 11/135 boost-mp11:x64-linux...
Elapsed time to handle boost-mp11:x64-linux: 2.1 ms
Installing 12/135 boost-variant2:x64-linux...
Elapsed time to handle boost-variant2:x64-linux: 1.07 ms
Installing 13/135 vcpkg-cmake-get-vars:x64-linux...
Elapsed time to handle vcpkg-cmake-get-vars:x64-linux: 984 us
Installing 14/135 boost-modular-build-helper:x64-linux...
Elapsed time to handle boost-modular-build-helper:x64-linux: 1.33 ms
Installing 15/135 boost-build:x64-linux...
Elapsed time to handle boost-build:x64-linux: 42.8 ms
Installing 16/135 boost-system:x64-linux...
Elapsed time to handle boost-system:x64-linux: 3.28 ms
Installing 17/135 boost-move:x64-linux...
Elapsed time to handle boost-move:x64-linux: 3.21 ms
Installing 18/135 boost-core:x64-linux...
Elapsed time to handle boost-core:x64-linux: 4.31 ms
Installing 19/135 boost-smart-ptr:x64-linux...
Elapsed time to handle boost-smart-ptr:x64-linux: 6.01 ms
Installing 20/135 boost-preprocessor:x64-linux...
Elapsed time to handle boost-preprocessor:x64-linux: 22.9 ms
Installing 21/135 boost-io:x64-linux...
Elapsed time to handle boost-io:x64-linux: 1.74 ms
Installing 22/135 boost-utility:x64-linux...
Elapsed time to handle boost-utility:x64-linux: 3.3 ms
Installing 23/135 boost-detail:x64-linux...
Elapsed time to handle boost-detail:x64-linux: 2.38 ms
Installing 24/135 boost-optional:x64-linux...
Elapsed time to handle boost-optional:x64-linux: 2 ms
Installing 25/135 boost-mpl:x64-linux...
Elapsed time to handle boost-mpl:x64-linux: 77.1 ms
Installing 26/135 boost-typeof:x64-linux...
Elapsed time to handle boost-typeof:x64-linux: 5.65 ms
Installing 27/135 boost-tuple:x64-linux...
Elapsed time to handle boost-tuple:x64-linux: 2.22 ms
Installing 28/135 boost-function-types:x64-linux...
Elapsed time to handle boost-function-types:x64-linux: 5.49 ms
Installing 29/135 boost-bind:x64-linux...
Elapsed time to handle boost-bind:x64-linux: 3.23 ms
Installing 30/135 boost-function:x64-linux...
Elapsed time to handle boost-function:x64-linux: 3.15 ms
Installing 31/135 boost-functional:x64-linux...
Elapsed time to handle boost-functional:x64-linux: 2.19 ms
Installing 32/135 boost-describe:x64-linux...
Elapsed time to handle boost-describe:x64-linux: 2.54 ms
Installing 33/135 boost-container-hash:x64-linux...
Elapsed time to handle boost-container-hash:x64-linux: 2.56 ms
Installing 34/135 boost-fusion:x64-linux...
Elapsed time to handle boost-fusion:x64-linux: 51 ms
Installing 35/135 boost-conversion:x64-linux...
Elapsed time to handle boost-conversion:x64-linux: 2.52 ms
Installing 36/135 boost-concept-check:x64-linux...
Elapsed time to handle boost-concept-check:x64-linux: 3.98 ms
Installing 37/135 boost-iterator:x64-linux...
Elapsed time to handle boost-iterator:x64-linux: 5.01 ms
Installing 38/135 boost-align:x64-linux...
Elapsed time to handle boost-align:x64-linux: 3.95 ms
Installing 39/135 boost-atomic:x64-linux...
Elapsed time to handle boost-atomic:x64-linux: 11.1 ms
Installing 40/135 boost-filesystem:x64-linux...
Elapsed time to handle boost-filesystem:x64-linux: 7.43 ms
Installing 41/135 boost-nowide:x64-linux...
Elapsed time to handle boost-nowide:x64-linux: 4.36 ms
Installing 42/135 vcpkg-cmake-config:x64-linux...
Elapsed time to handle vcpkg-cmake-config:x64-linux: 2.36 ms
Installing 43/135 fmt:x64-linux...
Elapsed time to handle fmt:x64-linux: 4.3 ms
Installing 44/135 hwloc:x64-linux...
Elapsed time to handle hwloc:x64-linux: 15.1 ms
Installing 45/135 ragel:x64-linux...
Elapsed time to handle ragel:x64-linux: 4.38 ms
Installing 46/135 pcre:x64-linux...
Elapsed time to handle pcre:x64-linux: 5.43 ms
Installing 47/135 boost-unordered:x64-linux...
Elapsed time to handle boost-unordered:x64-linux: 4.17 ms
Installing 48/135 boost-integer:x64-linux...
Elapsed time to handle boost-integer:x64-linux: 2.85 ms
Installing 49/135 boost-regex:x64-linux...
Elapsed time to handle boost-regex:x64-linux: 10.8 ms
Installing 50/135 boost-array:x64-linux...
Elapsed time to handle boost-array:x64-linux: 2.71 ms
Installing 51/135 boost-range:x64-linux...
Elapsed time to handle boost-range:x64-linux: 8.8 ms
Installing 52/135 boost-numeric-conversion:x64-linux...
Elapsed time to handle boost-numeric-conversion:x64-linux: 4.77 ms
Installing 53/135 boost-intrusive:x64-linux...
Elapsed time to handle boost-intrusive:x64-linux: 6.66 ms
Installing 54/135 boost-container:x64-linux...
Elapsed time to handle boost-container:x64-linux: 7.85 ms
Installing 55/135 boost-lexical-cast:x64-linux...
Elapsed time to handle boost-lexical-cast:x64-linux: 3.92 ms
Installing 56/135 boost-exception:x64-linux...
Elapsed time to handle boost-exception:x64-linux: 4.19 ms
Installing 57/135 boost-tokenizer:x64-linux...
Elapsed time to handle boost-tokenizer:x64-linux: 2.75 ms
Installing 58/135 boost-algorithm:x64-linux...
Elapsed time to handle boost-algorithm:x64-linux: 6.27 ms
Installing 59/135 boost-date-time:x64-linux...
Elapsed time to handle boost-date-time:x64-linux: 6.98 ms
Installing 60/135 boost-rational:x64-linux...
Elapsed time to handle boost-rational:x64-linux: 2.73 ms
Installing 61/135 boost-ratio:x64-linux...
Elapsed time to handle boost-ratio:x64-linux: 4.03 ms
Installing 62/135 boost-chrono:x64-linux...
Elapsed time to handle boost-chrono:x64-linux: 6.12 ms
Installing 63/135 boost-thread:x64-linux...
Elapsed time to handle boost-thread:x64-linux: 12.1 ms
Installing 64/135 boost-dynamic-bitset:x64-linux...
Elapsed time to handle boost-dynamic-bitset:x64-linux: 3.22 ms
Installing 65/135 boost-random:x64-linux...
Elapsed time to handle boost-random:x64-linux: 7.84 ms
Installing 66/135 boost-circular-buffer:x64-linux...
Elapsed time to handle boost-circular-buffer:x64-linux: 3.37 ms
Installing 67/135 boost-ptr-container:x64-linux...
Elapsed time to handle boost-ptr-container:x64-linux: 4.83 ms
Installing 68/135 boost-multi-array:x64-linux...
Elapsed time to handle boost-multi-array:x64-linux: 3.93 ms
Installing 69/135 boost-icl:x64-linux...
Elapsed time to handle boost-icl:x64-linux: 8.31 ms
Installing 70/135 boost-type-index:x64-linux...
Elapsed time to handle boost-type-index:x64-linux: 3.72 ms
Installing 71/135 boost-variant:x64-linux...
Elapsed time to handle boost-variant:x64-linux: 4.71 ms
Installing 72/135 boost-proto:x64-linux...
Elapsed time to handle boost-proto:x64-linux: 10.6 ms
Installing 73/135 boost-pool:x64-linux...
Elapsed time to handle boost-pool:x64-linux: 4.4 ms
Installing 74/135 boost-phoenix:x64-linux...
Elapsed time to handle boost-phoenix:x64-linux: 23.3 ms
Installing 75/135 boost-endian:x64-linux...
Elapsed time to handle boost-endian:x64-linux: 4.19 ms
Installing 76/135 boost-spirit:x64-linux...
Elapsed time to handle boost-spirit:x64-linux: 50.3 ms
Installing 77/135 boost-xpressive:x64-linux...
Elapsed time to handle boost-xpressive:x64-linux: 9.01 ms
Installing 78/135 boost-tti:x64-linux...
Elapsed time to handle boost-tti:x64-linux: 6.41 ms
Installing 79/135 boost-serialization:x64-linux...
Elapsed time to handle boost-serialization:x64-linux: 27.2 ms
Installing 80/135 boost-multi-index:x64-linux...
Elapsed time to handle boost-multi-index:x64-linux: 7.43 ms
Installing 81/135 boost-format:x64-linux...
Elapsed time to handle boost-format:x64-linux: 4.92 ms
Installing 82/135 boost-any:x64-linux...
Elapsed time to handle boost-any:x64-linux: 3.97 ms
Installing 83/135 boost-property-tree:x64-linux...
Elapsed time to handle boost-property-tree:x64-linux: 5.18 ms
Installing 84/135 boost-property-map:x64-linux...
Elapsed time to handle boost-property-map:x64-linux: 4.29 ms
Installing 85/135 boost-parameter:x64-linux...
Elapsed time to handle boost-parameter:x64-linux: 8.12 ms
Installing 86/135 boost-math:x64-linux...
Elapsed time to handle boost-math:x64-linux: 48.8 ms
Installing 87/135 boost-foreach:x64-linux...
Elapsed time to handle boost-foreach:x64-linux: 5.24 ms
Installing 88/135 boost-lambda:x64-linux...
Elapsed time to handle boost-lambda:x64-linux: 6.03 ms
Installing 89/135 boost-bimap:x64-linux...
Elapsed time to handle boost-bimap:x64-linux: 10.4 ms
Installing 90/135 boost-graph:x64-linux...
Elapsed time to handle boost-graph:x64-linux: 24.2 ms
Installing 91/135 boost-crc:x64-linux...
Elapsed time to handle boost-crc:x64-linux: 5.32 ms
Installing 92/135 hyperscan:x64-linux...
Building hyperscan[core]:x64-linux...
-- Using cached intel-hyperscan-v5.4.0.tar.gz.
-- Cleaning sources at /home/username/src/todds/vcpkg/buildtrees/hyperscan/src/v5.4.0-ec5872d107.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/username/src/todds/vcpkg/downloads/intel-hyperscan-v5.4.0.tar.gz
-- Applying patch 0001-remove-Werror.patch
-- Using source at /home/username/src/todds/vcpkg/buildtrees/hyperscan/src/v5.4.0-ec5872d107.clean
-- Configuring x64-linux
-- Building x64-linux-dbg
-- Building x64-linux-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /home/username/src/todds/vcpkg/downloads/tools/cmake-3.25.1-linux/cmake-3.25.1-linux-x86_64/bin/cmake --build . --config Release --target install -- -v -j17
    Working Directory: /home/username/src/todds/vcpkg/buildtrees/hyperscan/x64-linux-rel
    See logs for more information:
      /home/username/src/todds/vcpkg/buildtrees/hyperscan/install-x64-linux-rel-out.log

Call Stack (most recent call first):
  /home/username/src/todds/build/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /home/username/src/todds/build/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/hyperscan/portfile.cmake:22 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)


error: building hyperscan:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+hyperscan
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?title=[hyperscan]+Build+error&body=Copy+issue+body+from+%2Fhome%2Fusername%2Fsrc%2Ftodds%2Fbuild%2Fvcpkg_installed%2Fvcpkg%2Fissue_body.md

-- Running vcpkg install - failed
CMake Error at vcpkg/scripts/buildsystems/vcpkg.cmake:892 (message):
  vcpkg install failed.  See logs for more information:
  /home/username/src/todds/build/vcpkg-manifest-install.log
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/CMakeDetermineSystem.cmake:121 (include)
  CMakeLists.txt:7 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
```
Please also reference this additional log for additional details on the actual error: [install-x64-linux-rel-out.log](https://github.com/microsoft/vcpkg/files/11277837/install-x64-linux-rel-out.log)


### Additional context

Hello,

I am trying to build a project called ["todds"](https://github.com/joseasoler/todds) which is a CPU-based DDS encoder optimized for fast batch conversions with high encoding quality.

todds depends on hyperscan, so my actual logs are from me attempting to build by calling vcpkg to get the libs, from cmake.

I can reproduce the issue the issue by directly calling `./vcpkg install hyperscan` without using our manifest as well.

I believe that the current hyperscan vcpkg port version is 5.4.0 - you can reference [this issue](https://github.com/intel/hyperscan/issues/292) which you can see was fixed in the [hyperscan 5.4.1](https://github.com/intel/hyperscan/releases/tag/v5.4.1) release. 

The current hyperscan version is [hyperscan 5.4.2](https://github.com/intel/hyperscan/releases/tag/v5.4.2) which was released ~12 hours ago at the time of this Issue's submission.

Please update hyperscan vcpkg port to >= hyperscan 5.4.1 so that I can compile this project using the vcpkg dependency.

Thank you for your hard work!

--- a/ports/hyperscan/portfile.cmake
+++ b/ports/hyperscan/portfile.cmake
@@ -1,12 +1,10 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
-set(HYPERSCAN_VERSION 5.4.0)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO intel/hyperscan
-    REF v${HYPERSCAN_VERSION}
-    SHA512 cfec3f43b9e8b3fbb2e761927f3a173c1230f2688da710ec7708f2941ce6f550a1d3cb48b0b0e2ccf709807390117a7e40047cb99190bcc341f37eb3da13ae62
+    REF "v${VERSION}"
+    SHA512 328f21133161d16b36ebdc7f8b80a7afe7ca9e7e7433348e9bfa9acb5f3641522e8314beea1b219891f4e95f1392ff8036ebb87780fe808b8b4bd15a535e9509
     HEAD_REF master
     PATCHES
         0001-remove-Werror.patch
@@ -24,6 +22,6 @@ vcpkg_cmake_install()
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
-file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
 
 vcpkg_fixup_pkgconfig()

--- a/ports/hyperscan/vcpkg.json
+++ b/ports/hyperscan/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "hyperscan",
-  "version": "5.4.0",
+  "version": "5.4.2",
   "description": "A regular expression library with O(length of input) match times that takes advantage of Intel hardware to provide blazing speed.",
   "homepage": "https://www.hyperscan.io",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3129,7 +3129,7 @@
       "port-version": 0
     },
     "hyperscan": {
-      "baseline": "5.4.0",
+      "baseline": "5.4.2",
       "port-version": 0
     },
     "hypodermic": {

--- a/versions/h-/hyperscan.json
+++ b/versions/h-/hyperscan.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "174fd7953b696f08703b580901a840cc53157439",
+      "version": "5.4.2",
+      "port-version": 0
+    },
+    {
       "git-tree": "72c36aba3fff7cd403bdf02ad8f691ced9da30a9",
       "version": "5.4.0",
       "port-version": 0

